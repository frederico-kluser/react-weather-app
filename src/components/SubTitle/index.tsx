import SubTitleContainer from './styled';

interface SubTitleInterface {
  text: string;
}

const SubTitle = ({text}: SubTitleInterface) => (
  <SubTitleContainer>{text}</SubTitleContainer>
);

export default SubTitle;
