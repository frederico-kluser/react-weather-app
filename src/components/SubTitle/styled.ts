import styled from 'styled-components';
import { COLOR } from '../../utils/style';

const SubTitleContainer = styled.div`
  color: ${COLOR.gray};
  font-size: 1.34em;
  margin-bottom: 25px;
  margin-top: 5px;
`;

export default SubTitleContainer;
