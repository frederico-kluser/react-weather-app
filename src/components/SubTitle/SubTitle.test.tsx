import renderer from 'react-test-renderer';
import SubTitle from '.';

test('SubTitle Renderer', () => {
  const component = renderer.create(
    <SubTitle text="test" />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});