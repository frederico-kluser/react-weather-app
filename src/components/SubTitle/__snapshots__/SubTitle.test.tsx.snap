// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubTitle Renderer 1`] = `
<div
  className="sc-bdvvtL ljnmSX"
>
  test
</div>
`;
